app/elements/elements.html

```diff
@@ -22,3 +22,5 @@
 
 <!-- TODO(victorbalan): investigate why this element cause the flickering -->
 <link rel="import" href="/elements/skill/skill-list.html">
+
+<link rel="import" href="/elements/app-element.html">
```

Comment:
since the app element should be the root elements, it will be (in my opinion) always available and thus must not be in any "import common" files